src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1041,6 +1060,25 @@ IS 'foreign key from transcription_document';
 COMMENT ON COLUMN external_object_directory.ado_id
 IS 'foreign key from annotation_document';
 
+CREATE TABLE extobjdir_process_detail

```

Comment:
@jackmaloney This is a new table to hold 2 of the EOD fields for now. Though there is no Dev tickets yet to make the corresponding code in dart-api.

Reply:
Shall we just put a ticket on the backlog and leave Joe to prioritise it?